libs/directus/directus-query/project.json

```diff
@@ -27,6 +33,8 @@
           "mode": "production"
         }
       }
-    }
+    },
+    "poject-dry-release": {},
```

Comment:
Fix the typo on "project-dry-release" in all the files (the "r" from "project" is missing)